packages/react-components/react-shared-contexts/stories/UseAnnouce/UseAnnounceDefault.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  AnnounceProvider,
+  Button,
+  Divider,
+  Field,
+  Input,
+  Radio,
+  RadioGroup,
+  useAnnounce,
+} from '@fluentui/react-components';
+import type { AnnounceContextValue } from '@fluentui/react-components';
+
+const AnnounceMock: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+  const announce: AnnounceContextValue['announce'] = React.useCallback((message, options) => {
+    alert(`Announced {polite: ${String(options?.polite ?? false)}}: ${message}`);
+  }, []);
+  const value: AnnounceContextValue = React.useMemo(() => ({ announce }), [announce]);
+
+  return <AnnounceProvider value={value}>{children}</AnnounceProvider>;
+};
+
+const AnnouncePlayground: React.FC = () => {
+  const { announce } = useAnnounce();
+
+  const [message, setMessage] = React.useState('Hello world');
+  const [messageType, setMessageType] = React.useState<'polite' | 'assertive'>('polite');
+
+  return (
+    <>
+      <Field label="A message for annoucement">
+        <Input onChange={(ev, data) => setMessage(data.value)} value={message} />
+      </Field>
+      <Field label="Message type">
+        <RadioGroup onChange={(ev, data) => setMessageType(data.value as 'polite' | 'assertive')} value={messageType}>
+          <Radio label="assertive" value="assertive" />
+          <Radio label="polite" value="polite" />
+        </RadioGroup>
+      </Field>
+
+      <Button
+        onClick={() => {
+          announce(message, {
+            polite: messageType === 'polite',
+          });
+        }}
+      >
+        Announce message
+      </Button>
+    </>
+  );
+};
+
+export const Default = () => {
+  return (
+    <AnnounceMock>
```

Comment:
IMO the announce mock code should go in the main example body, it's fairly simple and we the goal of the doc is to show how to overload the announce context

Reply:
Indeed, I implemented it differently initially (playground was in body itself) and head chicken-egg problem with the context. I will update it.